### DISCUSSION TITLE
Редактирование полей из цикла в таблице товаров.

### DISCUSSION THREAD

**Discussion by Qb1cc** [totumonline/totum-ru-issues-and-discussions#108] [Как это сделать?] [answered]
Заметил проблему при добавлении из цикла в таблицу товаров. Первая строка добавляется нормально, и после добавления сразу сохраняется всё в поле. Но при последующих добавлениях, после добавления товара в колонку не падает добавленный товар, как будто пустой, но при этом в БД он сохраняется.
Если отключить индивидуальный расчёт селекта для строк, то начинает работать нормально , но автоматом не выбирает, приходится из всего списка выбирать добавленный товар. Если отключаю расчет селекта для каждой строки, то один раз всё автоматом выбирает, но последующие добавления выглядят вот так, пока не перезагрузишь страницу, товар не появляется даже в списке выбора, но при этом наименование подтягивает корректно, а поле артикул как будто бы пустое остаётся.

Как это починить, что бы после того как я добавил новый товар, он автоматом падал в строку с артикулом и всеми нужными колонками. 
![image](https://user-images.githubusercontent.com/84081305/163668499-39f1884a-3466-446e-97f4-7e72caa7e543.png)

**Comment by Qb1cc** [comment, marked as answer]
UPD: это ошибка она исправлена в 2.3.38.20, но надо помнить, что добавляемая строка в таблице источнике должна удовлетворять условиям выборки

Разобрался, нужно было выключить редактировать в WEB.

**Reply by ttm-alexeypolunin** [reply]
Я думаю у нас там ошибка в случае, когда используется индивидуальный расчет селекта.

Я сниму «ответ» и переведу в issue: https://github.com/totumonline/totum-ru-issues-and-discussions/issues/109

**Reply by ttm-alexeypolunin** [reply]
Готово в 2.3.38.20
